src/Workspaces/Core/Portable/CodeStyle/AnalyzerConfigOptionsExtensions.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Reflection;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis
+{
+    internal static class AnalyzerConfigOptionsExtensions
+    {
+        private readonly static MethodInfo _tryGetEditorConfigOptionMethodInfo = typeof(AnalyzerConfigOptionsExtensions).GetMethod("TryGetEditorConfigOption", BindingFlags.NonPublic | BindingFlags.Static);
+
+        public static bool TryGetOption<T>(this AnalyzerConfigOptions analyzerConfigOptions, Option<T> option, out T value)
+        {
+            return TryGetEditorConfigOption(analyzerConfigOptions, option, out value);
+        }
+
+        public static bool TryGetOption<T>(this AnalyzerConfigOptions analyzerConfigOptions, PerLanguageOption<T> option, string language, out T value)
+        {
+            return TryGetEditorConfigOption(analyzerConfigOptions, option, out value);
+        }
+
+        internal static bool TryGetOption(this AnalyzerConfigOptions analyzerConfigOptions, OptionKey optionKey, out object value)
```

Comment:
What is this needed for? I would have assumed everybody was using the slightly more strongly typed form below?